walker_test.go

```diff
@@ -35,6 +35,21 @@ import (
 	fspb "github.com/google/fswalker/proto/fswalker"
 )
 
+type testWriter struct {
+	outpath string
```

Comment:
or it could be just a string type to simplify a tiny bit but not a big deal at all:

```go
type outpathWriter string

func (o outpathWriter) writeWalk(walk *fspb.Walk) error {
  // ...
  return ioutil.WriteFile(o, walkBytes, 0444)
}
```

Reply:
Done, thanks